pkgs/servers/sql/mssql/server/default.nix

```
@@ -0,0 +1,122 @@
+{ stdenv
+, fetchurl
+, lib
+, rpmextract
+, makeWrapper
+, bash
+, python2
+, numactl
+, kerberos
+, e2fsprogs
+, sssd
+, libuuid
+, pam
+, openldap
+, systemd
+, libunwind
+, libredirect
+}:
+
+with lib;
+
+stdenv.mkDerivation rec {
+  pname = "mssql-server";
+  version = "15.0.4003.23-3";
+
+  src = fetchurl {
+    url = "https://packages.microsoft.com/rhel/7/mssql-server-2019/${pname}-${version}.x86_64.rpm";
+    sha256 = "2c617e905b553e32b37b48f2d3cbce9bc6274e1e2f56172ff9efd696d264bbbc";
+  };
+
+  dontUnpack = true;
+  nativeBuildInputs = [
+    makeWrapper
+    rpmextract
+  ];
+
+  ldLibraryPath = stdenv.lib.makeLibraryPath [
+    stdenv.cc.cc
+    numactl
+    kerberos
+    e2fsprogs
+    sssd
+    libuuid
+    pam
+    openldap
+    systemd.lib
+    libunwind
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    rpmextract $src
+    mkdir -p $out/bin
+    mv opt $out
+    mv usr $out
+    fix_bash() 
+    {
+      substituteInPlace $1 \
```

```suggestion
      substituteInPlace $@ \
```
and then you would be able to pass many file names:
```
fix_bash $out/opt/mssql/lib/mssql-conf/{invokesqlservr,checkinstall}.sh
```

---

Looking at the source, I don't think I can do that:
```
substituteInPlace() {
    local fileName="$1"
    shift
    substitute "$fileName" "$fileName" "$@"
}
```

---

Take a look at the latest version, I think it captures the spirit of this review.

---

I apologize, I really thought those helpers were able to take many arguments.

Don't really want to you churn on bike-shedding issues

---

Not a problem, the comment showed me things I didn't know about glob expansion.